src/xcoobee/api/EventsApi.js

```diff
@@ -73,6 +74,43 @@ const getEvents = (apiUrlRoot, apiAccessToken, userCursor, privateKey, passphras
     });
 };
 
+/**
+ *
+ * @param {string} apiUrlRoot - The root of the API URL.
+ * @param {ApiAccessToken} apiAccessToken - A valid API access token.
+ * @param {string} [campaignId] - Campaign id.
+ * @param {string} [type] - Event type to trigger.
+ *
+ * @returns {Promise<Object>} - The result.
+ * @property {Event[]} data - A page of events.

```

Comment:
These JSDocs are not accurate. Please update to reflect the data the promise resolves to.